modules/ocf/templates/brave/session-store-1.erb

```diff
@@ -1,23 +0,0 @@
-{
-   "about": {
-      "welcome": {
-         "showOnLoad": false
-      }
-   },
-   "settings": {
-      "advanced.pdfjs-enabled": false,
-      "general.check-default-on-startup": false,
-      "general.homepage": "<%= @browser_homepage %>",
-      "general.is-default-browser": false,
-      "general.startup-mode": "homePage",
-      "privacy.autofill-enabled": false,
-      "security.fullscreen.content": "alwaysAllow",
-      "security.passwords.active-password-manager": "Unmanaged",
-      "shutdown.clear-all-site-cookies": true,
-      "shutdown.clear-autocomplete-data": true,
-      "shutdown.clear-autofill-data": true,
-      "shutdown.clear-cache": true,
-      "shutdown.clear-downloads": true,
-      "shutdown.clear-history": true
```

Comment:
don't we still want some of these? are the equivalents in the new settings file?

Reply:
yes i want some (all) of these because i chose every setting here. these are no longer applicable and a lot don't have equivalents. the new settings GUI is also much harder to navigate. if you find a setting worth switching on lmk and i can add it. 

most the defaults have user privacy (default: ask user permission in all cases) in mind so it should be fine. the only thing i can think of adding that i've seen is clearing cookies on browser exit. disabling brave from asking to be the default browser is in the new config, there is no password manager because i opted not to install `brave-keyring`, so that should also take care of the autocomplete stuff.

we can have brave clear it's cache, downloads, history, etc on shutdown if we run it with the `--incognito` flag, but then the start screen is purple, so we don't get our homepage. it also seems a bit useless since `~/` will be deleted on user suspend, and even with all that stuff enabled, rootstaff could still get all this data if they poked around while someone was running a browser session.